src/components/charts/other.js

```diff
@@ -495,6 +552,70 @@ export default class Other extends Component {
     )
   }
 
+  renderPolarConfiguration() {
+    const { polar } = this.state.configurations;
+    return (
+      <div className="other-polar-container">
+        <div className="form-group config-option polar-type-selector">
+          <label>Type of the Chart:</label>
+          <select
+            className="form-control"
+            onChange={this.onPolarCheckBoxChange}
+            name={optionsPolar.chartType}>
+            <option>Line</option>
```

Comment:
This is the dangerous place... These labels are used in the code to change the configurations